packages/editor/src/editor/browser/Browser.tsx

```diff
@@ -25,11 +27,20 @@ export const Browser = ({ value, onChange, activeBrowsers, close, options }: Bro
   const attrBrowser = useAttributeBrowser(options);
   const logicBrowser = useLogicBrowser();
   const cmsBrowser = useCmsBrowser();
+  const conditionBuilder = useConditionBuilder({
+    generateConditionString: generateConditionString,
```

Comment:
if value is the same as key you don't have to write it twice
```suggestion
    generateConditionString,
```